cardano-cli/test/cardano-cli-golden/files/golden/governance/vote/voteView

```diff
@@ -0,0 +1,4 @@
+drep-keyhash-eb09d5556a8bce421074e394d02c79ced96741657b4cf7ca8995294d:
+  b1015258a99351c143a7a40b7b58f033ace10e3cc09c67780ed5b2b0992aa60a#5:
+    anchor: null
+    decision: VoteYes
```

Comment:
We should output `JSON` by default.  I like the idea of supporting `YAML` as an option.

Reply:
Should we drop the `anchor` field if its value is missing?